tests/cloner_test.go

```diff
@@ -121,3 +126,32 @@ func doCloneTest(f *framework.Framework, targetNs *v1.Namespace) {
 		Expect(err).ToNot(HaveOccurred())
 	}
 }
+
+func verifyTargetContent(f *framework.Framework, namespace *v1.Namespace, pvc *v1.PersistentVolumeClaim) bool {
+	By("Verify target PVC content matches source PVC")
+	executorPod, err := utils.CreateExecutorPodWithPVC(f.K8sClient, "verify-pvc-content", namespace.Name, pvc)
+	Expect(err).ToNot(HaveOccurred())
+	err = utils.WaitTimeoutForPodReady(f.K8sClient, executorPod.Name, namespace.Name, utils.PodWaitForTime)
+	Expect(err).ToNot(HaveOccurred())
+	output := f.ExecShellInPod(executorPod.Name, namespace.Name, "cat "+testFile)
+	f.DeletePod(executorPod)
+	return strings.Compare(fillData, output) == 0
+}
+
+func cloneOfAnnoExistenceTest(f *framework.Framework, targetNs *v1.Namespace) {
+	// Create targetPvc
+	By("Creating target pvc")
+	targetPvc, err := utils.CreatePVCFromDefinition(f.K8sClient, targetNs.Name, utils.NewPVCDefinition(
+		"target-pvc",
+		"1G",
+		map[string]string{controller.AnnCloneRequest: f.Namespace.Name + "/" + sourcePVCName, controller.AnnCloneOf: "true"},
+		nil))
+	Expect(err).ToNot(HaveOccurred())
+	err = utils.WaitForPersistentVolumeClaimPhase(f.K8sClient, targetNs.Name, v1.ClaimBound, targetPvc.Name)
```

Comment:
I don't think you can wait for ClaimBound, as with local volume that will not happen unless a pod attaches the PVC to itself. Which won't happen.

Reply:
So just merged a patch that should make it easier to inspect the controller log to see if it is skipping the PVC you are interested in.